src/ArcxAnalyticsSdk.ts

```diff
@@ -178,7 +178,7 @@ export class ArcxAnalyticsSdk {
   private _handleAccountDisconnected() {
     if (!this.currentChainId || !this.currentConnectedAccount) {
       this._report(
-        'error',
+        'warning',
```

Comment:
Add `TODO:`  comment with the task which tells us to handle edge case when it happens